SDK_versioned_docs/version-3.0.0/guides/liquidity/05-adding-liquidity.md

```diff
@@ -0,0 +1,29 @@
+---
+id: liquidity-actions
+title: Adding Liquidity
+---
+
+## Adding Liquidity to the Position
+
+Constructing the calldata for adding liquidity uses the same function call to `addCallParameters`, which takes in a `Position` and an options field of type `AddLiquidityOptions`. Since the goal of this example is to add liquidity to an already existing position, construct `IncreaseOptions` instead of `MintOptions`. `IncreaseOptions` types are defined by the interfaces `CommonAddLiquidityOptions & IncreaseSpecificOptions`. `CommonAddLiquidityOptions` is outlined above because minting also required these parameters and the reference for `IncreaseSpecificOptions` is:
```

Comment:
is `CommonAddLiquidityOptions & IncreaseSpecificOptions` intended to be one code block or two?  If two, suggest 'and' instead of '&'

Reply:
it's ok to leave I think. In the code, the interface is defined by the combination of these two interfaces (it uses an &)